test/MathOptInterface/MOI_Wrapper.jl

```diff
@@ -1,297 +1,188 @@
-        "solve_qcp_edge_cases",
-        "solve_qp_edge_cases",
-        "solve_qp_zero_offdiag",
-            
-        # These tests require extra parameters to obtain certificates.
-        "solve_farkas_equalto_upper",
-        "solve_farkas_equalto_lower",
-        "solve_farkas_lessthan",
-        "solve_farkas_greaterthan",
-        "solve_farkas_interval_upper",
-        "solve_farkas_interval_lower",
-        "solve_farkas_variable_lessthan",
-        "solve_farkas_variable_lessthan_max",
-       ],
-    )
-    # certificates
-    MOIT.solve_farkas_equalto_upper(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    MOIT.solve_farkas_equalto_lower(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    MOIT.solve_farkas_lessthan(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    MOIT.solve_farkas_greaterthan(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    MOIT.solve_farkas_interval_upper(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    MOIT.solve_farkas_interval_lower(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    MOIT.solve_farkas_variable_lessthan(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    MOIT.solve_farkas_variable_lessthan_max(BRIDGED_CERTIFICATE_OPTIMIZER,CONFIG)
-    # tolerance
-    MOIT.solve_qcp_edge_cases(BRIDGED_OPTIMIZER, CONFIG_LOW_TOL)
-    MOIT.solve_qp_edge_cases(BRIDGED_OPTIMIZER, CONFIG_LOW_TOL)
-    MOIT.solve_qp_zero_offdiag(BRIDGED_OPTIMIZER, CONFIG_LOW_TOL)
-    # MOIT.delete_soc_variables(OPTIMIZER, CONFIG_LOW_TOL)
-    MOIT.modificationtest(BRIDGED_OPTIMIZER, CONFIG)
+function runtests()
+    for name in names(@__MODULE__; all = true)
+        if startswith("$(name)", "test_")
+            @testset "$(name)" begin
+                getfield(@__MODULE__, name)()
+            end
+        end
+    end
 end
 
-@testset "Linear tests" begin
-    MOIT.contlineartest(
-        BRIDGED_OPTIMIZER,
-        MOIT.TestConfig(
-            basis = true,
-            infeas_certificates = false
-        ), [
-            # These tests require extra parameters to obtain certificates.
-            "linear8a", "linear8b", "linear8c","linear12",
-        ]
+function test_runtests()
+    optimizer = Xpress.Optimizer(OUTPUTLOG = 0)
+    model = MOI.Bridges.full_bridge_optimizer(optimizer, Float64)
+    MOI.set(model, MOI.Silent(), true)
+    MOI.Test.runtests(
+        model,
+        MOI.Test.Config(atol = 1e-3, rtol = 1e-3),
+        exclude = String[
+            "test_objective_set_via_modify",
+            "test_model_ListOfConstraintAttributesSet",
+            "test_solve_conflict_feasible",
+            "test_objective_get_ObjectiveFunction_ScalarAffineFunction",
+            "_SecondOrderCone_",
+            "test_constraint_PrimalStart_DualStart_SecondOrderCone",
+            "_RotatedSecondOrderCone_",
+            "_GeometricMeanCone_",
+            # It seems that Xpress cannot handle nonconvex quadratic constraint
+            "test_quadratic_nonconvex_",
+            # TODO: conflict tests should be investigated further
+            "test_solve_conflict_"

```

Comment:
These need investigating

Reply:
- Xpress does not accept nonconvex quadratic.
 
 - Xpress does not handle bounds properly in the IIS utility